src/libstd/sync/once.rs

```diff
@@ -114,15 +114,15 @@ impl Once {
         // otherwise we run the job and record how many people will try to grab
         // this lock
         let guard = self.mutex.lock();
-        if self.cnt.load(Ordering::SeqCst) > 0 {
+        if self.cnt.load(Ordering::Acquire) > 0 {
```

Comment:
This can be `Relaxed`: The acquisition of the result of the function is already done by the mutex.